bundles/specmate-integration-test/src/com/specmate/test/integration/NLPServiceTest.java

```diff
@@ -121,6 +128,69 @@ public void testMatcher() throws SpecmateException {
 
 	}
 
+	@Test
+	public void testDependencyParse() throws SpecmateException {

```

Comment:
so far, the OptionMatcher is not tested here, can you modiffy your test to also use the OptionMatcher

Can you also add tests for cases where no matches occur or only parts match?